### DISCUSSION TITLE
Why does my student certification application keep getting rejected？

### DISCUSSION THREAD

**Discussion by 0x6lack** [community/community#51665] [GitHub Education]
Why does my student certification application keep getting rejected?
I am a current student, I have also completed my personal details and am present at the school, provided an official student verification report and also have my school's education email address, but it keeps getting rejected for the following reasons:

> What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:
You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

Is there something wrong with your automatic detection mechanism?

**Comment by samus-aran** [comment]
Hi @0x6lack 👋 

Let's work through those rejection reasons:

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

It's stat that the campus location is not in your country. When you selected the campus/school in your application what location did it say it was in compared to the one you are currently attending...

In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

You will have the most success applying if you are able to do so from your school’s campus.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are using a VPN, please disable it when you reapply. 
- If you are not using a VPN, try reapplying when connected to the internet from a different network.

Hope this helps! Let us know how you get along 👍
